tests/phpunit/tests/media/shortcodes/test-wpml-page-builders-media-shortcodes-update.php

```diff
@@ -8,20 +8,6 @@ class Test_WPML_Page_Builders_Media_Shortcodes_Update extends \OTGS\PHPUnit\Tool
 	const ORIGINAL_POST_ID   = 9;
 	const TRANSLATED_POST_ID = 57;
 
-	/**
-	 * @test
-	 */
-	public function it_should_not_translate_if_content_has_no_shortcode() {
```

Comment:
@kagg-design, you can keep this test but you need to configure `WPML_Page_Builders_Media_Shortcodes`'s mock to return `false` for the method `has_media_shortcode`.

Reply:
As code is moved to `WPML_Page_Builders_Media_Shortcodes`, so test is moved to `Test_WPML_Page_Builders_Media_Shortcodes` and named `it_should_return_false_when_content_has_no_shortcode()`

Reply:
@strategio test added, please check